Unix/regress

```diff
@@ -278,7 +278,7 @@ fi
 ##
 
 rm -rf $prefix
-./configure --prefix=$prefix --outputdirname=$current_output $options
+./configure --enable-werror --prefix=$prefix --outputdirname=$current_output $options
```

Comment:
--enable-werror in this case catches some compilation problems you don't get with --dev, since it's not a DEBUG build.

If --enable-microsoft is for run-time only, it's probably not necessary to also enable --enable-werror.